Packs/SafeNet_Trusted_Access/Classifiers/classifier-SafeNet_Trusted_Access_–_Account_Lockout_Alert_Classifier.json

```diff
@@ -0,0 +1,68 @@
+{
+    "description": "Classifies SafeNet Trusted Access account lockout alerts.",
+    "feed": false,
+    "id": "accccf10-97da-425d-840f-beac0233efb1",
```

Comment:
```suggestion
    "id": "SafeNet Trusted Access – Account Lockout Alert Classifier",
```

Reply:
Done. Updated ID to classifier name.

![image](https://user-images.githubusercontent.com/86044314/140712322-a1b8107e-3f44-44e5-8064-c0539a845607.png)